docs/docs/aproc/aproc_doc.md

```diff
@@ -0,0 +1,309 @@
+# ARLAS Processes (APROC)
+
+ARLAS Processes (APROC) exposes an OGC API Processes compliant API.
+
+List of processes:
+
+- `ingest` : it ingests an archive.
+- `directory_ingest` : it ingests archives found in a directory.
+- `download` : it ingests an archive.
+- `enrich` : it enriches an item (like adding a cog).
+
+## Ingest process
+
+The `ingest` process takes a url pointing at an archive. The process runs the following steps:
+
+- identify the driver for ingestion
+- identify the assets to fetch (done by the driver)
+- fetch the assets (e.g. copy/download)  (done by the driver)
+- transform the assets if necessary (e.g. create cog)  (done by the driver)
+- upload the assets
+- register the item in AIRS
+
+As mentioned, the process is "driver" based. Each data source must have a compliant driver in order to be ingested in AIRS. A driver has to
+
+- say whether it supports a given archive or not
+- identify the archive's assets to be fetched
+- fetch the assets
+- transform the assets
+- create an AIRS Item
+
+A driver must implement the abstract class [Driver](https://github.com/gisaia/aias/blob/develop/extensions/aproc/proc/ingest/drivers/ingest_driver.py){:target="_blank"}.
+
+!!! warning
+    The name of the class within the module __must be__ `Driver`.
+
+The following drivers are available in the `extensions` directory:
+
+- ast_dem
+- digitalglobe
+- dimap
+- geoeye
+- rapideye
+- spot5
+- terrasarx
+- geotif and jpeg2000
+
+The drivers are configured in [drivers.yaml](https://github.com/gisaia/aias/blob/develop/conf/drivers.yaml){:target="_blank"}
+
+## Enrich process
+The `enrich` process takes a list of tuple collection/item id. The process runs the following step for each item:
+
+- say whether it supports a given archive or not
+- create the asset for the given item (done by the driver)
+- upload the asset
+- update the item
+
+A driver must implement the abstract class [Driver](https://github.com/gisaia/aias/blob/develop/extensions/aproc/proc/enrich/drivers/enrich_driver.py){:target="_blank"}.
+The following drivers are available in the `extensions` directory:
+
+- `safe` for sentinel 2 products
+
+The drivers are configured in [enrich_drivers.yaml](https://github.com/gisaia/aias/blob/develop/conf/enrich_drivers.yaml){:target="_blank"}
+
+## Using APROC
+
+### Prerequisites
+
+- python 3.10
+- AIRS
+- celery backend (redis)
+- celery brocker (rabbitmq)
+- docker
+
+See [here](https://hub.docker.com/r/gisaia/aproc-service/tags){:target="_blank"} for the available versions of aproc-service and [here](https://hub.docker.com/r/gisaia/aproc-proc/tags){:target="_blank"} for the available versions of aproc-processes
+
+### APROC Configuration
```

Comment:
this APROC Configuration should go to the project readme, not the doc that goes to docs.arlas.io